packages/components/src/ActionBar/ActionBar.scss

```diff
@@ -4,25 +4,48 @@ $tc-action-bar-margin: $padding-small $padding-larger $padding-small 0 !default;
 .tc-actionbar-container {
 	padding: 0 $padding-small;
 	background: $tc-action-bar-background-color;
-
+	height: 100%;
```

Comment:
this must be marked as a breaking change, so we have in our project to set the height of the container

Reply:
Done